### ISSUE TITLE
Install issue regarding pvlib version compatibility

### ISSUE THREAD

**Issue by toddkarin** [NREL/rdtools#121]
Tried a "pip install rdtools", but got back:
```
ERROR: pvfactors 1.1.0 has requirement pvlib>=0.6.0, but you'll have pvlib 0.5.2 which is incompatible.
```
It seems there is some requirement to have an earlier version of pvlib, is this necessary?

**Comment by mdeceglie** [comment]
Thanks for the heads up on that. The forthcoming versions of RdTools requires `pvlib >= 0.6.0, <0.7.0`. If you'd like to install that version now, you can get it in the development branch.

**Comment by mdeceglie** [comment]
related to #97

**Comment by cwhanse** [comment]
@mdeceglie what issues might need addressed for RdTools for pvlib >= 0.6.0?  pvlib 0.7.0 will have a number of API changes, I can volunteer to help RdTools with that as well.

**Comment by mdeceglie** [comment]
Thanks @cwhanse! The main issue we identified with moving up to pvlib>-0.6.0 was with the API change concerning the `perez_enhancement` keyword argument to   `loc.get_clearsky()`. I think we've got it covered in the development branch. But Will let you know if we run into further issues.
